packages/model/ColumnMapping.ts

```diff
@@ -9,6 +9,9 @@ import { initializer } from './lib/initializer'
 export interface ColumnMapping extends RdfResourceCore {
   sourceColumn: Link<CsvColumn>
   targetProperty: Term
+  datatype: NamedNode | null
+  language: string | null
+  defaultValue: Term | null
```

Comment:
Slightly less verbose option

```suggestion
  datatype?: NamedNode
  language?: string
  defaultValue?: Term
```

Reply:
These are not equivalent if I'm not mistaken 🤔